core/modules/memman/MemFile.h

```diff
@@ -48,15 +50,15 @@ class MemFile {
     //-----------------------------------------------------------------------------
```

Comment:
I think this doxygen comment block should be placed before the method declaration, otherwise it will be used as documentation for `struct MLResult`